src/main/java/view/InputView.java

```diff
@@ -0,0 +1,24 @@
+package view;
+
+import model.LadderHeight;
+import model.Names;
+
+import java.util.Scanner;
+
+public class InputView {
+    private Scanner scanner;
+
+    public InputView(Scanner scanner){
+        this.scanner = scanner;
+    }
+
+    public Names readPlayerNames(){
+        String names = scanner.nextLine();
+        return new Names(names);
+    }
+
+    public LadderHeight readLadderHeight(){
+        LadderHeight height = new LadderHeight(scanner.nextInt());
+        return height;
```

Comment:
요런 것은 바로 리턴해도 좋겠네요 ~ ! 
```suggestion
        return new LadderHeight(scanner.nextInt());
```